applications/nrf5340_audio/README.rst

```diff
@@ -419,78 +419,78 @@ LEDs
+| (bottom side LEDs around +-----------------------------------------------------------------------------------------------------------+
+| the center opening)      | Solid blue - The device is programmed as the left headset.                                                |
+|                          +-----------------------------------------------------------------------------------------------------------+
+|                          | Solid magenta - The device is programmed as the right headset.                                            |
+|                          +-----------------------------------------------------------------------------------------------------------+
+|                          | Solid yellow - The device is programmed with factory firmware.                                            |
+|                          | It must be re-programmed as gateway or headset.                                                           |
+|                          +-----------------------------------------------------------------------------------------------------------+
+|                          | Solid red (debug mode) - Fault in the application core has occurred.                                      |
+|                          | See UART log for details and use the **RESET** button to reset the device.                                |
+|                          | In the release mode, the device resets automatically with no indication on LED or UART.                   |
++--------------------------+-----------------------------------------------------------------------------------------------------------+
+| **RGB 2**                | Controlled by the Bluetooth LE Controller on the network core.                                            |
+|                          +-----------------------------------------------------------------------------------------------------------+
+|                          | Blinking green - Ongoing CPU activity.                                                                    |
+|                          +-----------------------------------------------------------------------------------------------------------+
+|                          | Solid red - Error.                                                                                        |
+|                          +-----------------------------------------------------------------------------------------------------------+
+|                          | Solid white (all colors on) - The **RGB 2** LED is not initialized by the Bluetooth LE Controller.        |
++--------------------------+-----------------------------------------------------------------------------------------------------------+
+| **ERR**                  | PMIC error or a charging error (or both). This LED also turns on when charging the battery over 7 hours.  |
```

Comment:
```suggestion
| **ERR**                  | PMIC error or a charging error (or both). This LED also turns on when charging the battery over 7 hours, since the PMIC has a timeout for protection.  |
```